src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java

```diff
@@ -7,18 +7,46 @@
 
 import junitparams.JUnitParamsRunner;
 import junitparams.Parameters;
+import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.runner.RunWith;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Address;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Venue;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
 
 @RunWith(JUnitParamsRunner.class)
 public class VenueDataLoaderTest {
 
-    private static final VenueDataLoader venueDataLoader = new VenueDataLoader();
+    private static final String VALID_EPIMS_ID = "45900";
```

Comment:
minor: would prefer to move this into the method if `venueDetailsBuilder` is no longer required